documentation/docs/cli/cli_prerequisites.md

```diff
@@ -0,0 +1,109 @@
+=== "Ubuntu 20.04"
+
+    Download package and install it:
+
+    ```console title="On local machine"
+    sudo apt update && sudo apt install -y wget
+    wget https://package.cosmian.com/kms/4.19.1/ubuntu-20.04/cosmian-kms-cli_4.19.1-1_amd64.deb
```

Comment:
keeping this version-specific is ok? Automatically incremented?

Reply:
Before releasing a new version, I replace all the occurences of X.Y.Z (could be better I know!)